core/controllers/editor_test.py

```diff
@@ -721,6 +726,12 @@ def add_logging_info(msg, *_):
             self.login(self.OWNER_EMAIL)
             self.testapp.delete(
                 '/createhandler/data/%s' % exp_id, expect_errors=True)
+
+            # Observed_log_messages[1] is 'Attempting to delete documents
+            # from index %s, ids: %s' % (index.name, ', '.join(doc_ids)) it is
```

Comment:
Add a full stop after the error message:

`doc_ids)).. It is logged by function`